SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestAttachedComponentReplication/SpatialTestAttachedComponentReplication.cpp

```diff
@@ -0,0 +1,218 @@
+﻿// Copyright 1998-2019 Epic Games, Inc. All Rights Reserved.
+
+#include "SpatialTestAttachedComponentReplication.h"
+
+#include "Algo/Copy.h"
+#include "Kismet/GameplayStatics.h"
+#include "Net/UnrealNetwork.h"
+#include "SpatialFunctionalTestStep.h"
+
+void USpatialTestAttachedComponentReplicationTestMap::CreateCustomContentForMap()
+{
+	Super::CreateCustomContentForMap();
+
+	AActor* LevelActor =
+		AddActorToLevel<ASpatialTestAttachedComponentReplicationActorForLevelPlacing>(World->GetCurrentLevel(), FTransform::Identity);
```

Comment:
very minor: could store the GetCurrentLevel in a local variable level to make subsequent lines a bit shorter